reproduceme/.htaccess

```diff
@@ -0,0 +1,48 @@
+Options +FollowSymLinks
+# Turn off MultiViews
+Options -MultiViews
+
+# Directive to ensure *.rdf files served as appropriate content type,
+# if not present in main apache config
+AddType application/rdf+xml .rdf
+AddType application/rdf+xml .owl
+AddType text/turtle .ttl
+AddType application/n-triples .n3
+AddType application/ld+json .json
+# Rewrite engine setup
+RewriteEngine On
+
+# Rewrite rule to serve HTML content from the REPRODUCE-ME vocabulary URI if requested
+RewriteCond %{HTTP_ACCEPT} !application/rdf\+xml.*(text/html|application/xhtml\+xml)
+RewriteCond %{HTTP_ACCEPT} text/html [OR]
+RewriteCond %{HTTP_ACCEPT} application/xhtml\+xml [OR]
+RewriteCond %{HTTP_ACCEPT} text/\* [OR]
+RewriteCond %{HTTP_ACCEPT} \*/\* [OR]
+RewriteCond %{HTTP_USER_AGENT} ^Mozilla/.*
+RewriteRule ^$ https://sheeba-samuel.github.io/REPRODUCE-ME/doc/index-en.html [R=303,L]
+
+# Rewrite rule to serve JSON-LD content from the REPRODUCE-ME vocabulary URI if requested
+RewriteCond %{HTTP_ACCEPT} application/ld+json
```

Comment:
This should be application/ld\+json (sorry, it's a typo in WIDOCO that will be addressed in the next release)